terraform/deployments/govuk-publishing-platform/frontends_rails_assets.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_s3_bucket" "frontends_rails_assets" {
+  bucket = "govuk-${var.govuk_environment}-${local.workspace}-frontends-rails-assets"
+
+  # Unless we're in staging or production, it's okay to delete this bucket
+  # even if it still contains objects.
+  #
+  # In the lower environments, we want to be able to run `terraform destroy`
+  # without having to manually delete all the assets from the bucket.
+  force_destroy = contains(["staging", "production"], var.govuk_environment) ? false : true
+
+  tags = {
+    name            = "govuk-${var.govuk_environment}-${local.workspace}-frontends-rails-assets"
+    aws_environment = var.govuk_environment
+  }
+}
+
+data "aws_iam_policy_document" "frontends_cloudfront_access_s3_policy" {
```

Comment:
I don't get quite what you mean, it is creating the policy document rather than `pulling`. See ref: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document

Reply:
Ah right! I wasn't aware that this data source creates a JSON, and doesn't pull anything from AWS. We've just created raw JSON policy docs elsewhere in the project.